backend/tsconfig.json

```diff
@@ -0,0 +1,55 @@
+{
+  // This attribute will hold all the compile option of TypeScript
+  "compilerOptions": {
```

Comment:
What are your thoughts on having a stricter compiler? IMO it'll probably be a bit annoying in the start, but we will get better code from it.

Reply:
Sure , do you have any suggestions?

Reply:
I added the following fields to the tsconfig.json

```json
    "strict": true,
    "alwaysStrict": false,
```

I guess we could add even more strict options, but i think it's a good place to start.

Reply:
Yeah, as long as you can't get away with only writing JS we're on the right track